### PR TITLE
Refactor VIP Dashboard to not use React

### PR DESCRIPTION
**Pull request by pauarge**
## Description

The VIP Dashboard on `wp-admin` was originally intended to contain many features, but currently is simply a bunch of links and a form. For that reason, using React and a whole build system might be a bit overkill. 

This PR gets rid of React and the `gulp` build system, from which we were using very old versions. It replaces the pages for plain HTML, CSS and JS.

With this change, the functionality remains intact but we don't have to bother with CI or merging endless dependency upgrade PRs.

### Refactor VIP Dashboard to not use React

The VIP dashboard now has a way simpler implementation and keeps all its functionality.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [x] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

1. Check out PR.
2. Go to `wp-admin` > VIP Dashboard
3. Try to send a test ticket

### CODE REVIEW

File: vip-dashboard/components/widget-welcome.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+				<li>
+					<a href="https://docs.wpvip.com/" target="_blank">VIP Documentation</a>
+					<span>Launching and developing with VIP</span>
+				</li>
+				<li>
+					<a href="https://wordpressvip.zendesk.com/" target="_blank">VIP Support Portal</a>
+					<span>Your organization’s tickets</span>
+				</li>
+				<li>
+					<a href="https://docs.wpvip.com/technical-references/vip-support/general-ticket-guidelines/" target="_blank">Ticket guidelines</a>
```
Comment (vaurdan) [inline vip-dashboard/components/widget-welcome.php:27]: Change this URL to https://docs.wpvip.com/technical-references/vip-support/#h-submitting-a-ticket

File: vip-dashboard/components/widget-welcome.php [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+					<span>How to open the perfect ticket</span>
+				</li>
+			</ul>
+		</div>
+
+		<div class="widget__col-2">
+			<ul class="widget__list">
+				<li>
+					<a href="https://docs.wpvip.com/how-tos/launch-a-site-with-vip/" target="_blank">Guidebook: Launching with VIP</a>
```
Comment (vaurdan) [inline vip-dashboard/components/widget-welcome.php:36]: Change URL to https://docs.wpvip.com/how-tos/launch-a-site/

File: vip-dashboard/components/widget-welcome.php [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+					<span>Steps to launch</span>
+				</li>
+				<li>
+					<a href="https://wpvip.com/documentation/developing-with-vip/welcome-to-vip-development/" target="_blank">Guidebook: Developing with VIP</a>
```
Comment (vaurdan) [inline vip-dashboard/components/widget-welcome.php:40]: Change URL to https://docs.wpvip.com/technical-references/development-workflow/. 

Perhaps we can also change the copy to something like "Development Workflow on VIP"?

File: vip-dashboard/components/widget-welcome.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+					<span>An overview of VIP development</span>
+				</li>
+				<li>
+					<a href="https://wpvip.com/news/" target="_blank">VIP News</a>
```
Comment (vaurdan) [inline vip-dashboard/components/widget-welcome.php:44]: Change URL to https://wpvip.com/blog/ and copy to "VIP Blog"

File: vip-dashboard.php [hunk 5]
```diff
@@ -2,11 +2,11 @@
...
  * Plugin Name: VIP Dashboard
- * Plugin URI: https://wpvip.com
+ * Plugin URI: http://wpvip.com
  * Description: WordPress VIP Go Dashboard
- * Author: Scott Evans, Filipe Varela, WPVIP
- * Version: 2.1.1
- * Author URI: https://wpvip.com
+ * Author: Scott Evans, Filipe Varela, Pau Argelaguet
+ * Version: 3.0.0
+ * Author URI: http://wpvip.com
```
Comment (vaurdan) [inline vip-dashboard.php:9]: Is there a particular reason for changing it to http?

**Review by vaurdan — APPROVED** [review]
Thank you for working on this :) Overall, the changeset looks good to me. I added some comments on old URLs that we can update since we're already touching this code. Since these are only suggestions and the code looks good, I'm approving it.

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
Please address @vaurdan's feedback

**Comment by rinatkhaziev** [comment]
@vaurdan thanks for the review! it may be worth using suggestions in cases like these so that it would be easier to apply the suggestions.

File: vip-dashboard.php (hunk 5 above)
Comment (pauarge) [reply vip-dashboard.php:9]: I was copy pasting. Indeed, it is better to have it with `https`.

**Comment by pauarge** [comment]
@vaurdan Thanks for your feedback, it makes way more sense to have the updated links.

**Comment by GaryJones** [comment]
I think things like updating URLs, and making strings internationalized, improving accessibility, etc. can be deferred until the main PR has gone in to remove React. With a simplified codebase, others could chip in easier then.

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**